### DISCUSSION TITLE
Make a way to move an item from complete back onto the board.

### DISCUSSION THREAD

**Discussion by canadaduane** [ckipp01/skan#36] [Feature Request]
I pressed `enter` on something "in progress" by mistake. How do I get it back?

**Comment by ckipp01** [comment]
Hey @canadaduane, good question. Currently you can't. If you accidentally hit `enter` on something in `TODO` and it moves to `In Progress` you can just hit `BACKSPACE` on it, but if it goes off the board currently you're sort of screwed.

🤔 I'm thinking maybe there should be a way to view all of your "Completed" items on a screen with some stats or something that would provide you a way to bump it back onto the board.

**Reply by poleguy** [reply]
No, you're not screwed. 

I did a grep and I see the data is stored in a json file in:
~/.local/share/skan/contexts/default.json

So the workaround is to simply go in there and change the status from DONE to TODO.

This is the very first thing I did on the very first issue I created in skan after trying it for the first time.
I thought, wow, does it simply delete completed items? Even on a real physical kanban board you can uncrumple the note from the trash and put it back on the board. Or at least read it to retype it.

I know this is a minimalist tool, but using enter for 'progress' and also for 'delete' is not awesome for discoverability.

Ideally you would have a key 'd' or 'delete' to mark it done and just have enter do nothing when in the "In Progress" column. 

'u' for undo of the last item marked done might be easy to implement?

A note in the readme about where the jason is stored would be nice too.

**Reply by poleguy** [reply]
I looked for this tool after trying nullboard and deciding I didn't like it storing my data in the browser cache on a single computer. This doesn't really solve the problem completely, but it's far better because at least I can ssh in to the machine that has my data if I'm away from it.

**Comment by poleguy** [comment]
In addition, pressing 'x' literally deletes the item from the json file. Then you really are screwed. Maybe good to have an undo on this as well, or mark it as DELETED status in the json rather than actually deleting it.
